domains/client/domain-operator/src/bundle_processor.rs

```diff
@@ -19,7 +19,20 @@ use sp_keystore::KeystorePtr;
 use sp_messenger::MessengerApi;
 use sp_runtime::traits::{Block as BlockT, Zero};
 use sp_runtime::{Digest, DigestItem};
+use sp_weights::constants::WEIGHT_REF_TIME_PER_MILLIS;
 use std::sync::Arc;
+use std::time::Instant;
+
+// The slow log threshold for consensus block preprocessing
+const SLOW_PREPROCESS_MILLIS: u64 = 500;
+
+// The slow log threshold for domain block execution: `reference_duration_ms * 1.2 + 200ms`
+fn slow_domain_block_execution_threshold(reference_duration_ms: u64) -> u64 {
+    reference_duration_ms + (reference_duration_ms / 5) + 200
```

Comment:
can you better comments explaining what this calculation is doing and how we are using to define slowness